.pipelines/templates/aks-setup.yaml

```diff
@@ -13,7 +13,6 @@ steps:
         -n ${AZURE_CLUSTER_NAME} \
         --subscription ${SUBSCRIPTION_ID} \
         --node-count ${AGENT_COUNT} \
-        --kubernetes-version ${AKS_KUBERNETES_VERSION} \
```

Comment:
Removing this so we can run the latest default for AKS. Without this we would need to update the `AKS_KUBERNETES_VERSION` if the current version is out of support in AKS.